lang/en.yml

```diff
@@ -149,7 +187,7 @@ en:
     HelpChars: ' Special characters are automatically converted or removed.'
     OK: OK
   SilverStripe\CMS\Model\RedirectorPage:
-    CLASS_DESCRIPTION: 'Redirects to an internal page or an external URL'
+    CLASS_DESCRIPTION: 'Redirects requests to another location'
```

Comment:
The "new" value here is the value it already had in the code in CMS 5. The value must have been manually altered in transifex to a different (and therefore wrong) value.